pkg/sentry/server/validator/internal/common.go

```diff
@@ -64,3 +64,17 @@ func namespaceIsRequired(namespace string) error {
 	}
 	return nil
 }
+
+// IsControlPlaneComponent returns true if the app ID corresponds to a Dapr control plane component.
+// Note: callers must additionally validate the namespace to ensure it matches the one of the Dapr control plane.
+func IsControlPlaneComponent(appID string) bool {
```

Comment:
```suggestion
// IsControlPlaneApp returns true if the app ID corresponds to a Dapr control plane component.
// Note: callers must additionally validate the namespace to ensure it matches the one of the Dapr control plane.
func IsControlPlaneApp(id string) bool {
```

`App` is semantically probably the right word to be using here.

Reply:
I agree with Component not being good. I'll use "Service" however because with "App" there's ambiguity as we normally talk about "apps" in the context of user code.